data/blog/2024/2024-07-23-current-maintenance-status-of-laminas-mezzio-packages.md

```diff
@@ -0,0 +1,38 @@
+---
+id: 2024-07-23-current-maintenance-status-of-laminas-mezzio-packages
+author: julian
+title: 'Current Maintenance Status of Laminas & Mezzio Packages'
+draft: false
+public: true
+created: '2024-07-23T11:00:00-01:00'
+updated: '2024-07-23T11:00:00-01:00'
+tags:
+  - laminas api tools
+  - custom proprieties
+  - maintenance status
+---
+
+Laminas Project has created a large number of packages to serve the needs of the PHP community. Unfortunately, there is
+no aggregated place to see the status of each package easily. You would need to visit each one of them on GitHub and
+check out the *Custom Properties* page. And that is when you remember to do so...
+
+<!--- EXTENDED -->
+
+### Laminas and Mezzio packages maintenance status at a glance
+
+The page below intends to provide a fast, accessible way to examine every package at a glance. It provides a column for
+each of the three organizations: Laminas, Api Tools and Mezzio.
+The page is automatically refreshed daily and is publicly available.
+
+[**Current Maintenance Status of Laminas & Mezzio Packages**](https://getlaminas.org/packages-maintenance-status/)
+
+There is also a JSON version [**here**](https://getlaminas.org/share/properties.json). It contains the full list of
```

Comment:
This information can already be found on the overview page, but it is not so important that it needs to be included in the blog post.

Reply:
fixed,  the article was refactored a bit